deploy/docker/passwd

```diff
@@ -0,0 +1 @@
+nobody:x:65534:65534:nobody:/home:/bin/false
```

Comment:
`nobody:x:65534:65534:nobody:/nonexistent:/usr/sbin/nologin` is more preferable. `nobody` should not be able to login.